coralproject/talk
# [CORL-1277] Replace fetch for axios on perspective requests

## What does this PR do?

Swaps out Perspective's createFetch requests with axios requests that natively support time outs. 

If this mitigates issues of failed requests and doesn't fail in other ways, this means we can switch over to this version of time out based fetch requests and call it a day.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

Locally:
- setting up perspective with an API key
- creating some bad comments
- see they get flagged and notify you when posting

In production:

- watch and see if we get reduced perspective network errors over time
